genanki/__init__.py

```diff
@@ -278,7 +278,7 @@ def __init__(self, deck_or_decks=None):
     else:
       self.decks = deck_or_decks
 
-  def write_to_file(self, file):
+  def write_to_file(self, file, media_files=[]):
```

Comment:
Please match the style of the rest of the library.

`media_files` should be an attribute on `Package` objects that can either be passed to `__init__` or set later. E.g.:

```python
my_package = Package(my_deck, media_files=['foo.jpg'])
```
or
```python
my_package = Package(my_deck)
my_package.media_files = ['foo.jpg']
```
or
```python
my_package = Package(my_deck)
my_package.media_files.append('foo.jpg')
```